### PR TITLE
change epoch used as sdk.Int to int64 in deposits.

### PR DESCRIPTION
**Pull request by puneet2019**
remove Antipattern: deposits stores epoch as sdk.Int rather than int64.

### CODE REVIEW

File: x/liquidstakeibc/keeper/deposit.go [hunk 1]
```diff
@@ -12,12 +12,12 @@ import (
 func (k *Keeper) SetDeposit(ctx sdk.Context, deposit *liquidstakeibctypes.Deposit) {
 	store := prefix.NewStore(ctx.KVStore(k.storeKey), liquidstakeibctypes.DepositKey)
 	bytes := k.cdc.MustMarshal(deposit)
-	store.Set([]byte(deposit.ChainId+deposit.Epoch.String()), bytes)
+	store.Set(append([]byte(deposit.ChainId), sdk.Uint64ToBigEndian(uint64(deposit.Epoch))...), bytes)
```
Comment (kruspy) [inline x/liquidstakeibc/keeper/deposit.go:15]: Move this to `keys.go`

File: x/liquidstakeibc/keeper/deposit.go [hunk 2]
```diff
@@ -12,12 +12,12 @@ import (
...
 }
 
 func (k *Keeper) DeleteDeposit(ctx sdk.Context, deposit *liquidstakeibctypes.Deposit) {
 	store := prefix.NewStore(ctx.KVStore(k.storeKey), liquidstakeibctypes.DepositKey)
-	store.Delete([]byte(deposit.ChainId + deposit.Epoch.String()))
+	store.Delete(append([]byte(deposit.ChainId), sdk.Uint64ToBigEndian(uint64(deposit.Epoch))...))
```
Comment (kruspy) [inline x/liquidstakeibc/keeper/deposit.go:20]: Same here

File: x/liquidstakeibc/keeper/deposit.go [hunk 3]
```diff
@@ -28,12 +28,12 @@ func (k *Keeper) CreateDeposits(ctx sdk.Context, epoch int64) {
...
 			ChainId:       hc.ChainId,
 			Amount:        sdk.NewCoin(hc.IBCDenom(), sdk.NewInt(0)),
-			Epoch:         sdk.NewInt(epoch),
+			Epoch:         epoch,
 			State:         liquidstakeibctypes.Deposit_DEPOSIT_PENDING,
 			IbcSequenceId: "",
 		}
 		bytes := k.cdc.MustMarshal(deposit)
-		store.Set([]byte(deposit.ChainId+deposit.Epoch.String()), bytes)
+		store.Set(append([]byte(deposit.ChainId), sdk.Uint64ToBigEndian(uint64(deposit.Epoch))...), bytes)
```
Comment (kruspy) [inline x/liquidstakeibc/keeper/deposit.go:36]: Same here

File: x/liquidstakeibc/keeper/deposit.go (hunk 1 above)
Comment (puneet2019) [reply x/liquidstakeibc/keeper/deposit.go]: used formatInt as you have used in other functions, and moved it

**Review by kruspy — APPROVED** [review]

**Pull request merged**